app/src/main/res/values/strings.xml

```diff
@@ -454,6 +454,10 @@
     <string name="customize_toggle_recent_searches" moz:removedIn="96" tools:ignore="UnusedResources">Recent searches</string>
     <!-- Title for the customize home screen section with Pocket. -->
     <string name="customize_toggle_pocket">Pocket</string>
+    <!-- Title for preference to open wallpapers -->
+    <string name="preferences_wallpapers">Wallpapers</string>
+    <!-- Content description for various wallpapers -->
+    <string name="content_description_wallpaper_name">Wallpaper Item: %1$s</string>
```

Comment:
There's no formal structure to this, but observing just how our strings are organized - I think we want to flip this around to be `wallpaper_name_content_description`. I am not totally aware of the full context (mostly just reviewing based on style/consistency right now) but I am also wondering if `name` is the right description for the string name given the context.

Also, another aside about how strings are (sometimes) organized, it would be ideal to group related strings under a `<!-- Wallpaper -->` grouping if it makes sense, but otherwise, just prefixing the string based on the feature can be very helpful to finding related strings.

Reply:
On another note, I think `content_description_wallpaper_name ` is actually used within the Wallpaper fragment. So, it totally makes sense to create `<!-- Wallpaper -->` grouping instead of keeping it under `Customize home`.

Reply:
Looks like a pretty mixed bag of `content_description` being a suffix or prefix. I'll defer to your suggestion here and update though. Does `wallpaper_settings_item_content_description` make more sense?

Reply:
Ya sounds fine, I just wasn't sure if `name` quite described the true intent given the `Wallpaper Item: %1$s` context that I was working with.

Reply:
Cool. Updating with these suggestions

Reply:
Also make sure to explain what `%1$s` gets replaced with in the comment above. The comments will show up in the translation tool (Pontoon) and if this is missing then the L10N team will block the string import until its resolved. :)